Cargo.toml

```
@@ -110,7 +110,7 @@ libp2p-request-response = { version = "0.24.0", path = "protocols/request-respon
 libp2p-swarm = { version = "0.42.0", path = "swarm" }
 libp2p-uds = { version = "0.37.0", path = "transports/uds", optional = true }
 libp2p-wasm-ext = { version = "0.39.0", path = "transports/wasm-ext", optional = true }
-libp2p-yamux = { version = "0.43.0", path = "muxers/yamux", optional = true }
+libp2p-yamux = { version = "0.43.0-alpha", path = "muxers/yamux", optional = true }
```

```suggestion
libp2p-yamux = { version = "0.43.0", path = "muxers/yamux", optional = true }
```

I suggest releasing a stable version right away.